include/k4arecord/playback.hpp

```diff
@@ -288,11 +288,11 @@ class playback
 
     /** Get the last valid timestamp in the recording
      *
-     * \sa k4a_playback_get_last_timestamp_usec
+     * \sa k4a_playback_get_recording_length_usec
      */
-    std::chrono::microseconds get_last_timestamp() const noexcept
+    std::chrono::microseconds get_recording_length() const noexcept
```

Comment:
The C++ API for playback has not been released yet, so this change should be fine without worrying about deprecation.